Release 1.12 plan.md

```diff
@@ -0,0 +1,33 @@
+# Plan for Version 1.12
+
+## Timing
+
+The last feature release of chakracore occurred on 29th June 2018 (version 1.10) since then there have been numerous security update releases and 1 non-security bug fix release (1.11). There has however been significant feature work performed since that date all of which is unreleased.
+
+We would therefore like our first release to be done fairly quickly - however there are also some half complete pieces of work to finish before that as well as a variety of bugs introduced that should be fixed before that.
+
+We will aim to do this first release by June to allow time for confirming the exact status of the project, completing unfinished work, performing testing and fixing bugs.
+
+## Actions prior to release
+
+We will endeavour to perform the below work prior to issuing this first community release in June. If time permits additional work may be done - conversely if time is too short the release may be produced with some of these actions not taken - in either event what has and has not been done will be explained.
+
+1. Review major changes between 1.11 and master to generate an accurate set of release notes.
+
+1. Review current master against standards to identify major gaps including assessing against the tc39 list of stage 3 and stage 4 proposals and also testing with test262. This action may identify further actions to be taken before release including fixing small deviations from the standards.
+
+1. Review all open bug reports and assess impact, fix where practicable.
+
+1. Complete partially complete work including:
+
```

Comment:
I am aware that there is also a partially complete BigInt implementation but I have no idea how far along it is - hence not put it in the list of things to complete for the first release though perhaps it could be a stretch goal?

Reply:
Found #5440. How important that is vs regex below? I can take a look into what those two things would take to do, though this does look a lower-hanging fruit. @zenparsing your input would also be appreciated :)

We don't have to sort this plan out in here, maybe should open a separate issue.

Reply:
The current BigInt implementation is not very far along - perhaps 1/3 complete. I think that the current partial implementation needs quite a bit of refactoring as well.

I think BigInt is pretty important. If we can reuse or port some existing OSS code for arbitrary precision integer arithmetic then it shouldn't be too difficult.

Reply:
I try to setup a periodic call between OSS contributors and former Chakra people, so that you have a medium to surface questions like this.